Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -358,14 +363,16 @@ function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, b
 
     %% 7.   Labeling efficiency
     if ~isfield(x.Q,'LabelingEfficiency') || isempty(x.Q.LabelingEfficiency)
-        if ~isfield(x.Q,'LabelingEfficiency')
-            switch lower(x.Q.LabelingType)
-                case 'pasl'
-                    x.Q.LabelingEfficiency = 0.98; % (concensus paper, Wong, MRM 1998)
-                case 'casl'
-                    x.Q.LabelingEfficiency = 0.85; % (concensus paper, Dai, MRM 2008)
-            end
-        end
+		switch lower(x.Q.LabelingType)
+			case 'pasl'
+				if x.MagneticFieldStrength == 7
```

Comment:
For new variables I would use the same name as BIDS uses, is that the case here? Otherwise fine